inject-generator/src/test/java/io/avaje/inject/generator/InjectProcessorTest.java

```diff
@@ -41,6 +41,7 @@ void deleteGeneratedFiles() {
     }
   }
 
+  @Disabled
```

Comment:
The module import gets enabled ... and that does not work against Valhalla EA. So maybe thinking of not using that module import feature.

Reply:
can we have this test disabled only for valhalla?

Reply:
Yeah maybe. The origin issue as I see it is the "module import" feature ... it gets turned on due to --enable-preview but it isn't in the Valhalla EA build so that fails. So for me it makes more sense to disable that when using Valhalla or just disable it for now until the Valhalla EA also has that feature included.

You don't perfer that idea?